eslint.config.js

```diff
@@ -0,0 +1,19 @@
+import eslintConfigPrettier from 'eslint-config-prettier'
+import js from '@eslint/js'
+import globals from 'globals'
+
+export default [
+  js.configs.recommended,
+  eslintConfigPrettier,
+
+  {
+    rules: {
+      'no-console': ['error', { allow: ['warn', 'error'] }],
```

Comment:
What do y'all think about preventing console.log() from being committed to codebases? 

Seems like `js.configs.recommended` will catch `debugger` statements. 

This current rule lets you put warn & error into the logs.

Reply:
I think if `warn` and `error` are allowed, then `console.log()` shouldn't be, since `warn` and `error` are usually meant to document important information that could be helpful to fix bugs on staging/production environments, `log` is primarily used for debugging/testing, and shouldn't be committed.

Reply:
Correct! This rule works like you expect.

The `no-console` rule blocks all uses of `console`. But adding `{ allow: ['warn', 'error'] }` means that we can use those without error.